### PR TITLE
Resize too large images in TinyMce image selection

### PR DESCRIPTION
**Pull request by ngilain**

### CODE REVIEW

**Comment by nicolasThal** [comment]
:+1:

File: MediaAdminBundle/Resources/views/BackOffice/Underscore/Include/previewImageView._tpl.twig [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<img id="preview_thumbnail" src="<%= src %>" alt="image preview"/>
+<img id="preview_thumbnail" src="<%= src %>" alt="image preview"/><br />
```
Comment (veolys-nanne) [inline MediaAdminBundle/Resources/views/BackOffice/Underscore/Include/previewImageView._tpl.twig]: use display:block to replace <br />

**Comment by ngilain** [comment]
@itkg-nanne : updated

File: MediaAdminBundle/Resources/public/less/media.less [hunk 2]
```diff
@@ -78,17 +78,26 @@ img:not([src]) {
...
+  width:250px;
 }
 
 .media_crop_preview, .media_format_actions {
-    margin-bottom:15px;
+  margin-bottom:15px;
+}
+
+img#preview_thumbnail {
+  max-width:745px;
```
Comment (alavieille) [inline MediaAdminBundle/Resources/public/less/media.less]: i think is better to use a max-width in percent `max-width:100%`

**Comment by ngilain** [comment]
@itkg-alavieille : update done

**Comment by alavieille** [comment]
:+1:

**Pull request merged**